trecs/models/recommender.py

```diff
@@ -456,7 +456,8 @@ def run(
         random_items_per_iter=0,
         vary_random_items_per_iter=False,
         repeated_items=True,
-    ):
+        no_new_items=False,
```

Comment:
Easy switch to turn content creation on/off (something that will be helpful for my experiments)